webdriver-spec.html

```diff
@@ -5602,33 +5605,54 @@
   of <a>trying</a> to <a>get a known element</a>
   with argument <var>element id</var>.
 
+ <li><p><a>Scroll into view</a> the <var>element</var>.
```

Comment:
I may just change the definition of `scroll into view` to handle the case where the element is already in view.